### PR TITLE
IF-476: add creation date to att + field

### PR DESCRIPTION
**Pull request by g1glh0rn**

### CODE REVIEW

**Comment by mattmikulina** [comment]
Task linked: [IF-476 Column Date created of Metrics & Attributes List is hard coded and not part of the field object](https://app.clickup.com/t/2177764/IF-476)

File: fusionbackend/src/main/java/io/fusion/fusionbackend/dto/mappers/AssetTypeTemplateMapper.java [hunk 1]
```diff
@@ -50,6 +50,7 @@ private AssetTypeTemplateDto toDtoShallow(final AssetTypeTemplate entity) {
                 .publicationState(entity.getPublicationState())
                 .publishedDate(entity.getPublishedDate())
                 .publishedVersion(entity.getPublishedVersion())
+                .creationDate(entity.getCreationDate())
```
Comment (jsylupp) [inline fusionbackend/src/main/java/io/fusion/fusionbackend/dto/mappers/AssetTypeTemplateMapper.java:53]: add creationDate also in toEntity()

File: fusionbackend/src/main/java/io/fusion/fusionbackend/dto/mappers/FieldMapper.java [hunk 2]
```diff
@@ -48,6 +48,7 @@ private FieldDto toDtoShallow(Field entity) {
                 .unitId(EntityDtoMapper.getEntityId(entity.getUnit()))
                 .thresholdType(entity.getThresholdType())
                 .widgetType(entity.getWidgetType())
+                .creationDate(entity.getCreationDate())
```
Comment (jsylupp) [inline fusionbackend/src/main/java/io/fusion/fusionbackend/dto/mappers/FieldMapper.java:51]: same here

File: fusionbackend/src/main/java/io/fusion/fusionbackend/dto/mappers/UnitMapper.java [hunk 3]
```diff
@@ -47,7 +47,7 @@ private UnitDto toDtoShallow(Unit entity) {
                 .name(entity.getName())
                 .label(entity.getLabel())
                 .symbol(entity.getSymbol())
-                .createdDate(entity.getCreatedDate())
+                .creationDate(entity.getCreationDate())
```
Comment (jsylupp) [inline fusionbackend/src/main/java/io/fusion/fusionbackend/dto/mappers/UnitMapper.java:50]: same here

File: fusionbackend/src/main/java/io/fusion/fusionbackend/dto/AssetTypeTemplateDto.java [hunk 4]
```diff
@@ -37,6 +38,7 @@
     private PublicationState publicationState;
     private OffsetDateTime publishedDate;
     private Long publishedVersion;
+    private Date creationDate;
```
Comment (jsylupp) [inline fusionbackend/src/main/java/io/fusion/fusionbackend/dto/AssetTypeTemplateDto.java:41]: Is Date or OffsetDateTime to be used (on both occasions)?

**Review by jsylupp — CHANGES_REQUESTED** [review]
Sieht soweit gut aus. Bei toEntity bitte noch ergänzen ;)

**Pull request merged**